oeps/processes/oep-0001.rst

```diff
@@ -415,6 +432,9 @@ OEP.
 * Additional diagrams or clarifying material (as long as the `Architecture
   Group`_ agrees that the substance of the OEP isn't changed).
 
+For the latter two points, please follow `Step 4. Announcing Changes`_ to inform
```

Comment:
I’m not clear on exactly what “the latter two points” refers to. Instead, could we say “If any changes are made…”, or something like that?